synfig-studio/src/gui/app.cpp

```diff
@@ -3615,7 +3615,7 @@ void App::open_vectorizerpopup(const etl::handle<synfig::Layer_Bitmap> my_layer_
 {
 	String desc = my_layer_bitmap->get_description();
 	synfig::info("Opening Vectorizerpopup for :"+desc);
-	App::vectorizerpopup = new studio::VectorizerSettings(*App::main_window,my_layer_bitmap,selected_instance,vectorizer_configmap,reference_layer);
+	App::vectorizerpopup = studio::VectorizerSettings::create(my_layer_bitmap,selected_instance,vectorizer_configmap,reference_layer);
 	App::vectorizerpopup->show();
```

Comment:
You need to check `App::vectorizerpopup` for null value.

Reply:
What should the error dialog look like if the value is null?

Reply:
You are already catching errors, so you know what happened: the glade file was not found, or a builder constructor error, etc.
To display the error dialog, you can use `App::dialog_message_1b` method.

Reply:
Thanks!

Reply:
Just move the Glade file outside and try to open the dialog and see what happens.